packages/manager/modules/pci/src/projects/project/training/training.constants.js

```diff
@@ -3,10 +3,17 @@ export const COMMUNITY_URL = 'https://community.ovh.com/en/c/Data-AI';
 export const DISCORD_URL = 'https://discord.com/invite/vXVurFfwe9';
 export const DOC_DOCKER_BUILD_URL =
   'https://docs.ovh.com/gb/en/ai-training/build-use-custom-image/';
+export const JOB_SSH_KEYS_CONSTANTS = {
```

Comment:
i think we can avoid the '_CONSTANTS' word suffix, we are already into a const file.

Reply:
fixed